common/web/lm-worker/build.sh

```diff
@@ -51,23 +51,23 @@ wrap-worker-code ( ) {
   # but also adds a lot more code the worker doesn't need to use.
   # Recommended by MDN while keeping the worker lean and efficient.
   # Needed for Android / Chromium browser pre-41.
-  cat "../../../node_modules/string.prototype.codepointat/codepointat.js" || die
+  cat "../../../node_modules/string.prototype.codepointat/codepointat.js" || builder_die
```

Comment:
OK, so it _is_ used here... but if we have a `set -e` on this script, I think we can just remove its references here and let `set -e` handle it?

It'd be simple enough to test by eliminating the `cat`-ed file, at least.

Reply:
Reverted